src/graphql/gqls/entry.ts

```diff
@@ -0,0 +1,11 @@
+import { gql } from 'graphql-request';
+
+export const ENTRY_COUNT = gql`
+  query TranscendCliEntryCount(
+    $filterBy: UnstructuredSubDataPointRecommendationsFilterInput
+  ) {
+    unstructuredSubDataPointRecommendations(filterBy: $filterBy) {
```

Comment:
make sure to pass useMaster: false

Reply:
ya definitely recommend - can be added in a second PR. but that will help alleviate the load on our db. given the amount of data here can be a ton its pretty important

Reply:
https://github.com/transcend-io/main/pull/32251